config/locales/client.en.yml

```diff
@@ -3554,7 +3554,7 @@ en:
 
       flag_topic:
         title: "Flag"
-        help: "privately flag this topic for attention or send a private notification about it"
```

Comment:
I think we do want to make clear that flags are not public. How would you change this to make that clear without using the word "privately" here? I agree with changing "private notification" to "personal message".